### PR TITLE
Documentation Creation

### PR DESCRIPTION
**Pull request by jordan-rash**
- Created Github Action for GitHub Pages
- Started MkDocs documentation migration

Note: a lot of this code is boiler plate Mkdocs code

### CODE REVIEW

**Review by claymcleod — CHANGES_REQUESTED** [review]
I don't think the `site/` directory needs to be committed. The built site should only be available on the `gh-pages` branch.

**Comment by claymcleod** [comment]
I can see it's working as intended here: https://github.com/jordan-rash/CICERO/tree/gh-pages. Maybe just remove that directory and add `site/` to the `.gitignore`.

**Comment by jordan-rash** [comment]
Good catch.  Cleaned up all the tag alongs

**Review by claymcleod — APPROVED** [review]
LGTM

**Comment by jordan-rash** [comment]
@adthrasher @claymcleod I don't have write access to merge this. It looks like a recent commit has formed a conflict.  Ill get that sorted and then push the update.

**Comment by adthrasher** [comment]
Thanks @jordan-rash. I merged an outstanding PR earlier and that's probably what caused the conflict. Let me know when you're ready and I can do the merge.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,73 @@
...
+          src="https://img.shields.io/badge/License-MIT-blue.svg" />
+   </a>
+  </p>
+
+
+  <p align="center">
+    CICERO (Clipped-reads Extended for RNA Optimization) is an assembly-based algorithm to detect diverse classes
+    of driver gene fusions from RNA-seq.
+   <br />
+   <a href="https://stjudecloud.github.io/cicero/"><strong>Explore the docs »</strong></a>
```
Comment (adthrasher) [inline README.md:35]: This is actually under the `stjude` org and not the `stjudecloud` org.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,73 @@
...
+   <br />
+   <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02043-x"><strong>Read the paper »</strong></a>
+   <br />
+   <br />
+   <a href="https://github.com/stjudecloud/cicero/issues/new?assignees=&labels=&template=feature_request.md&title=Descriptive%20Title&labels=enhancement">Request Feature</a>
```
Comment (adthrasher) [inline README.md:40]: Same as above

File: README.md [hunk 3]
```diff
@@ -1,40 +1,73 @@
...
+   <a href="https://github.com/stjudecloud/cicero/issues/new?assignees=&labels=&template=bug_report.md&title=Descriptive%20Title&labels=bug">Report Bug</a>
```
Comment (adthrasher) [inline README.md:41]: Same as above

**Review by adthrasher — APPROVED** [review]

**Comment by jordan-rash** [comment]
@adthrasher This should be read to go.  Can be viewed here: https://jordan-rash.github.io/CICERO/

**Pull request merged**